LibreNMS/Util/Rewrite.php

```diff
@@ -112,6 +112,8 @@ public static function normalizeIfName($name)
     public static function shortenIfName($name)
     {
         $rewrite_shortif = [
+            'hundredgige' => 'Hu',
+            'twentyfivegige' => 'Tw',
```

Comment:
Isn’t this suppose to be TF ?

Reply:
On Cisco devices I work with, "Tw" is twentyFive. I don't have a device with both 25G and 2.5G to check unfortunately. We can put "Twe" if it removes a doubt and is more consistant between vendors. At least, for Cisco, `Tw`and `Twe` are the same from the CLI point of view ...

Reply:
I definitely saw some documents point to TF too :)

Reply:
Also have these fun ones: ce, xe, et

Just pick whatever you think best represents it :)

Reply:
Let move to `Twe` then -> it is the first 3 letters so it makes sense in the CLI point of view, and it does not collide like `Tw` did with 2.5. I'll update the PR.